docs/installation/mac.md

```diff
@@ -12,15 +12,16 @@ title: macOS
    [rbenv](https://github.com/rbenv/rbenv). Please follow their
    [installation guide](https://github.com/rbenv/rbenv#installation).
 2. With the Ruby version manager, install the Ruby version listed on our badge.
-   (i.e. with rbenv: `rbenv install $(cat .ruby-version)`)
+   (i.e. with rbenv: `rbenv install *latest badge version*`) 
+    example: `rbenv install 2.7.1`
```

Comment:
``(i.e. with rbenv: `rbenv install $(cat .ruby-version)`)`` should stay as it is reading the `.ruby-version` file from the forked Forem repository.

If this happens before the docs mention grabbing the repository, we should rework that too.

Reply:
Yes it is happening before there is mention of forking the repo but we can mention to fork the repository first instead and keep `(i.e. with rbenv: `rbenv install $(cat .ruby-version)`)`